app/src/main/res/layout/item_skill.xml

```diff
@@ -51,15 +51,15 @@
                     android:layout_gravity="center"
                     android:layout_weight="1"
                     android:background="@color/md_white_1000"
+                    android:fontFamily="sans-serif"
                     android:gravity="center"
                     android:maxLines="3"
                     android:padding="@dimen/padding_small"
                     android:textAlignment="center"
                     android:textColor="@color/md_grey_700"
                     android:textSize="@dimen/text_size_normal"
                     android:textStyle="italic"
-                    tools:text="@string/sample_website_description"
-                    android:fontFamily="sans-serif"/>
+                    tools:text="@string/sample_website_description" />
```

Comment:
unrelated change here, you just have moved the lines from one position to another

Reply:
The actual work was already done in earlier commits which is mentioned in pr description but I forget to reformat the code so I used Ctrl + Alt + L